modular_skyrat/modules/airlock_override/code/airlock_override.dm

```diff
@@ -149,6 +149,27 @@ GLOBAL_VAR_INIT(force_eng_override, FALSE)
 		SEND_GLOBAL_SIGNAL(COMSIG_GLOB_FORCE_ENG_OVERRIDE, FALSE)
 		return
 
+/proc/eng_override_on(auto = FALSE)
```

Comment:
having an `on`, `off`, and `toggle` proc feels unnecessary

Reply:
oh yeah, I have a signal for that